samples/word/50-document/insert-external-document.yaml

```diff
@@ -71,7 +89,12 @@ template:
                 <br>
```

Comment:
With the expanded scope, consider changing the preceding label from
"Select a Word document to copy its body text into the current document."
to
"Select a Word document."

(I don't think you need to spell out why you're selecting a file, since that's mentioned two other places in the sample.